spec/pacio_test_kit/create_test_spec.rb

```diff
@@ -0,0 +1,118 @@
+RSpec.describe PacioTestKit::CreateTest do
+  let(:runnable) do
+    Class.new(PacioTestKit::CreateTest) do
+      config(
+        options: {
+          resource_type: 'Observation',
+          profile: 'PFESingleObservation'
+        }
+      )
+
+      input :url
+
+      fhir_client do
+        url :url
+      end
+    end
+  end
+  let(:session_data_repo) { Inferno::Repositories::SessionData.new }
+  let(:results_repo) { Inferno::Repositories::Results.new }
+  let(:test_session) { repo_create(:test_session, test_suite_id: 'pacio_pfe_server') }
+  let(:url) { 'https://example/r4' }
+  let(:resource_type) { 'Observation' }
+  let(:resource_id) { '123' }
+  let(:observation) do
+    JSON.parse(
+      File.read(File.join(__dir__, '..', 'fixtures', 'pfe_single_observation.json'))
+    )
+  end
+  # let(:resource_header) { Inferno::Entities::Header }
+  # let(:resource_header_params) {resource_header.id = 'example_id', resource_header.vid = 'example_vid', }
```

Comment:
```ruby
let(:response_headers) do
  { 'Location' => "#{url}/#{resource_type}/#{resource_id}" }
end
```
This will assign `{ 'Location' => 'https://example/r4/Observation/123' }` to `response_headers`